lang/ko.json

```diff
@@ -354,13 +354,13 @@
     },
     "events": {
       "exp": "{POINTS}경험치",
-      "availableUntilDate": "Available Until\n{DATE}",
+      "availableUntilDate": "\n{DATE}까지 유효함",
```

Comment:
![image](https://user-images.githubusercontent.com/3542488/163675376-2d570805-bffc-45fa-bf38-e5db0a3ca20e.png)
It looks like this the way you have it. Can you utilize the line break somehow to save space?